diff --git a/node_modules/@typescript/vfs/dist/vfs.cjs.development.js b/node_modules/@typescript/vfs/dist/vfs.cjs.development.js
index 2d19b7a..26a090b 100644
--- a/node_modules/@typescript/vfs/dist/vfs.cjs.development.js
+++ b/node_modules/@typescript/vfs/dist/vfs.cjs.development.js
@@ -164,11 +164,12 @@ var addAllFilesFromFolder = function addAllFilesFromFolder(map, workingDir) {

   var fs = require("fs");

-  var walk = function walk(dir) {
+  var walk = function walk(dir, filter = () => false) {
     var results = [];
     var list = fs.readdirSync(dir);
     list.forEach(function (file) {
       file = path.join(dir, file);
+        if (filter(file)) return;
       var stat = fs.statSync(file);

       if (stat && stat.isDirectory()) {
@@ -185,13 +186,24 @@ var addAllFilesFromFolder = function addAllFilesFromFolder(map, workingDir) {
   var allFiles = walk(workingDir);
   allFiles.forEach(function (lib) {
     var fsPath = "/node_modules/@types" + lib.replace(workingDir, "");
-    var content = fs.readFileSync(lib, "utf8");
     var validExtensions = [".ts", ".tsx"];

     if (validExtensions.includes(path.extname(fsPath))) {
-      map.set(fsPath, content);
+        var content = fs.readFileSync(lib, "utf8");
+        map.set(fsPath, content);
     }
   });
+    const cwd = path.resolve(__dirname, `../../../../../`);
+    const pkgName = require(path.resolve(cwd, 'package.json')).name
+    walk(cwd, (x) => x.match(/site|\.git|node_modules/)).forEach(function (lib) {
+        var fsPath = "/node_modules/" + pkgName + lib.replace(cwd, "").replace(/\\/g, "/");
+        var validExtensions = [".ts", ".tsx", ".json"];
+
+        if (validExtensions.includes(path.extname(fsPath))) {
+            var content = fs.readFileSync(lib, "utf8");
+            map.set(fsPath, content);
+        }
+    });
 };
 /** Adds all files from node_modules/@types into the FS Map */

